apps/gitness/src/pages/pull-request-list-page.tsx

```diff
@@ -34,6 +34,8 @@ const SortOptions = [
   { name: 'Updated', value: 'updated' }
 ] as const satisfies DropdownItemProps[]
 
+const colorArr = ['mint', 'yellow', 'red', 'blue', 'purple']
```

Comment:
we should be using theme colors instead of html colors

Reply:
![image](https://github.com/user-attachments/assets/f193a9b2-1cbd-412e-bcd3-9e961ba7dcc9)
so those colors were just mappings that joe had created in the playground component to give the theme to the badges